### PR TITLE
Feature/TS-9 build create account screen

### PR DESCRIPTION
**Pull request by AnoshaKhan**
####  I built create account screen

### CODE REVIEW

File: ShopGo/logInViewController.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @IBOutlet weak var phoneNoTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var nextButton: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        nextButton.layer.cornerRadius = 22
+        nextButton.clipsToBounds = true
+        nextButton.layer.masksToBounds = false
+        nextButton.layer.shadowColor = UIColor.gray.cgColor
```
Comment (haroonmaqsoodch) [inline ShopGo/logInViewController.swift:23]: create a method and move all the code in that and then call it in viewDidLoad()

**Review by haroonmaqsoodch — COMMENTED** [review]
Your Pr format isn't correct. Check other Prs and follow the format

File: ShopGo/logInViewController.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        nextButton.layer.shadowOffset = CGSize(width: 3, height: 3)
+        nextButton.layer.shadowRadius = 10
+        nextButton.layer.shadowOpacity = 1.0
+        
+//        let userNameImage = UIImage(named: "")
```
Comment (haroonmaqsoodch) [inline ShopGo/logInViewController.swift:28]: remove comments

**Review by haroonmaqsoodch — COMMENTED** [review]
You made changes on the main storyboard. Move your UIViewController to a separate storyboard file.

File: ShopGo/logInViewController.swift (hunk 2 above)
Comment (AnoshaKhan) [reply ShopGo/logInViewController.swift:28]: you didn't send me the logos of text fields , that's why I commented them otherwise it gives me fatal error.

**Pull request closed without merging**